sr_hand_config/2186/controls/host/common/friction_compensation.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+sr_friction_map: [
+#["LFJ5", 1,  [[0.0, 0.0], [1.0, 0.0]]],
+#["LFJ5", -1, [[0.0, -80.0], [1.0, -80.0]]]
```

Comment:
Is this meant to be commented out?

Reply:
i see in other hands too https://github.com/shadow-robot/sr_hand_config/blob/master/sr_hand_config/4138/controls/friction_compensation.yaml

Reply:
I think we should think of a cleaner way of doing this

Reply:
it should just be empty i think

Reply:
@shadow-robot/production_shadow any comment on this?